hooks/persistence-elastic/hook.test.js

```diff
@@ -5,28 +5,28 @@ beforeEach(() => {
   elasticClient.bulk.mockClear();
 });
 
+const scan = {
+  metadata: {
+    uid: "09988cdf-1fc7-4f85-95ee-1b1d65dbc7cc",
+    name: "demo-scan",
+    labels: {
+      company: "iteratec",
+    },
+  },
+  spec: {
+    scanType: "Nmap",
+    parameters: ["-Pn", "localhost"],
+  },
+};
+
+const now = new Date('2020-11-11');
```

Comment:
Great idea to pin this to a fixed date 👍

Reply:
Yes but should have updated the name of the variable too^^